Todo_app/urls.py

```diff
@@ -13,10 +13,12 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django import urls
 from django.contrib import admin
 from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('',include('app.urls') )
+    path('',include('app.urls') ),
+    path('',include('task.urls'))
```

Comment:
there should be any prefix as per functionality like :  /user/

Reply:
fixed